microservices-connector/config/samples/gmcmanager/gmc-manager-rbac.yaml

```diff
@@ -29,6 +29,7 @@ rules:
   resources:
```

Comment:
Can we move 

- gmc-manager-rbac.yaml to config/rbac  
- gmc-manager.yaml to config/manager
- gmcrouter as a sub dir under config?

Reply:
Hi @irisdingbj, I think we can discuss it. Basically, the yaml files, including `gmc-manager-rbac.yaml` and `gmc-manager.yam`, are sample for ChatQnA, that's why I move them into the folder of `sample`

Reply:
done!